gips/data/merra/merra.py

```diff
@@ -24,6 +24,15 @@
 import os
 import datetime
 import time
+
+from gips.data.merra.install_basic_client import install_basic_client
+
+URI = "urs.earthdata.nasa.gov"
+USER = "bobbyhbraswell"
+PASSWD = "Coffeedog_2"
```

Comment:
Yes that should be deleted. Thanks. Auth creds for this commit are in a .netrc file.